### ISSUE TITLE
Use a single `wires` keyword in `MultiControlledX`

### ISSUE THREAD

**Issue by albi3ro** [PennyLaneAI/pennylane#1679]
Currently, the operation [`MultiControlledX`](https://github.com/PennyLaneAI/pennylane/blob/6d93cc580d4e8860a0b3fc8cc42e81198399e665/pennylane/ops/qubit/non_parametric_ops.py#L858) accepts an arbitrary length of wires for `control_wires` keyword and a single wire for the `wires` keyword:

```
qml.MultiControlledX(control_wires=(0,1,2,3), wires=4)
```

We should instead be able to create this gate with on a single set of wires, where the last wire is the target with an `X` applied to it, similar to the syntax of `qml.CNOT` and `qml.Toffoli`:

```
qml.MultiControlledX(wires=(0,1,2,3,4))
```

We would have to implement this change with a deprecation cycle where we accept either syntax, but raise a `UserWarning` if the user provides the `control_wires` keyword.

**Comment by josh146** [comment]
Just tagging @trbromley to find out if there was a reason for the original design!

**Comment by trbromley** [comment]
I believe the original choice of `control_wires` and `wires` was just to mirror [ControlledQubitUnitary](https://github.com/PennyLaneAI/pennylane/blob/v0.16.0/pennylane/ops/qubit.py#L2112), which [MultiControlledX](https://github.com/PennyLaneAI/pennylane/blob/v0.16.0/pennylane/ops/qubit.py#L2236) inherited from at the time. Given that `MultiControlledX` now inherits from `Operation`, I don't see a blocker to implementing as outlined above.

**Comment by tgag17** [comment]
Hi! Can I work on this issue?

**Comment by antalszava** [comment]
Hi @tgag17, sure! :100: Let us know if you have any questions. :slightly_smiling_face:

**Comment by MoritzWillmann** [comment]
hey @tgag17, are you still working on this?

**Comment by natestemen** [comment]
I believe this should now be closed since #2121 is merged.

**Comment by josh146** [comment]
Nice catch @natestemen, thanks!
